analysis/analysis.py

```diff
@@ -28,6 +29,87 @@ def load_data_csv(csv_path: Path):
     return out_data
 
 
+def get_sentiments() -> dict:
+    """
+    Returns a dictionary of sentiment scores, with the keys being the word and the values being
+    their score
+
+    :return: dict mapping words to their sentiment scores
+    """
+    sentiment_path = Path('data', 'sentiments.txt')
+
+    sentiments = dict()
+    with open(sentiment_path, 'r') as file:
+        word = file.readline()
+
+        # We want to handle each word individually, rather than as a whole set
+        while word is not '':
```

Comment:
This can just be ```while word``` because the empty string is falsy (i.e., evaluates to false when bool() is called on it)

p.s. pylint is complaining about this line bc it's bad practice to use ```is``` to compare a literal (in this case the empty string) against an object. You'll want to use ```!=``` instead. This is because you can have an object with the same value as a literal that is not literally the same reference, which'll evaluate to false when you probably want true.